### PR TITLE
Add object ID decoding support

### PR DESCRIPTION
**Pull request by ThePlasmaRailgun**
Adds an optional boolean argument to the Decoder class of whether to decode object IDs into their common names. E.G. read() == '1.2.840.113549.1.1.5' to read() == 'sha1WithRSAEncryption'.

### CODE REVIEW

File: src/oids.py [hunk 1]
```diff
@@ -0,0 +1,4169 @@
+# From Peter Guttman at https://www.cs.auckland.ac.nz/~pgut001/dumpasn1.cfg
```
Comment (ThePlasmaRailgun) [inline src/oids.py]: I can include the program I used to process Peter's tables into a Python dictionary if you'd like.

**Comment by ThePlasmaRailgun** [comment]
Fixed the CI error

**Comment by andrivet** [comment]
This is an interesting new feature. But I see that you have not modified the unit tests. Can you please add a unit test for your new features ? Tests are in `tests`. Thanks.

**Comment by ThePlasmaRailgun** [comment]
Ok, I'll take a look.

**Comment by ThePlasmaRailgun** [comment]
I might not have time, I've got a lot of projects right now. Thanks.

**Comment by andrivet** [comment]
After some implementation (unit tests and some fixes in the code), I come to the conclusion that it is not right to introduce such features inside the parser. It should be done outside, as it is in the `dump.py` example.

**Pull request closed without merging**